test/correctness/clone_cache_key.cpp

```diff
@@ -0,0 +1,50 @@
+#include "Halide.h"
```

Comment:
It would be more descriptive if this filename contained "memoize" e.g. "memoize_cloned.cpp".